### DISCUSSION TITLE
flet build apk in windows 10

### DISCUSSION THREAD

**Discussion by Zahraa-sy** [flet-dev/flet#3765] [Q&A]
### Question
system windows 10 pro
command "flet build apk"
when i run the command i got this error

### Code sample

```python
import flet as ft


def main(page: ft.Page):
    page.add(ft.SafeArea(ft.Text("Hello, Flet!")))


ft.app(main)
```

### Error message

```bash
[19:08:27] Because myapp depends on wakelock_plus any which doesn't exist (authorization failed), version solving failed.

           Insufficient permissions to the resource at the https://pub.dev package repository.
           You can modify credentials using:
               dart pub token add https://pub.dev


[19:08:31] Error building Flet app - see the log of failed command above.
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Provide more info and context:  which OS, command you run.. etc

**Reply by Zahraa-sy** [reply]
I update the question and wrote the details.

**Reply by Zahraa-sy** [reply]
I found out that wakelock_plus is not present in flutter version that I downloaded ,and flet command "flet create apk" depends on it .So I think I need to flutter version include wakelock_plus
